hudi-flink/src/main/java/org/apache/hudi/sink/StreamWriteFunction.java

```diff
@@ -659,8 +659,8 @@ private void flushRemaining(boolean isEndInput) {
         .taskID(taskID)
         .instantTime(currentInstant)
         .writeStatus(writeStatus)
-        .isLastBatch(true)
-        .isEndInput(isEndInput)
+        .lastBatch(true)
+        .endInput(isEndInput)
```

Comment:
`isEndInput` -> `endInput`?

Reply:
No need to change that because `StreamWriteFunction` is not a POJO.

Reply:
I mean, you removed all the prefix "is" of the "isEndInput" from the repository except the getter of `WriteMetadataEvent`.  So, why not change this variable?

Reply:
Because `WriteMetadataEvent` is a POJO, we should respect the name pattern of the POJO, while there is no need to change for `StreamWriteFunction`.

Reply:
Fine, have changed it.

Reply:
I know `WriteMetadataEvent` is a POJO. I mean, let's follow one style.